src/plugins/slowed/index.tsx

```
@@ -0,0 +1,140 @@
+            <span class="sr-title"> SLOWED</span>
+          </div>
+          
+          <Show when={!collapsed()}>
+            <div class="sr-body">
+              <div class="sr-presets">
+                <button class="sr-btn" onClick={() => { setSpeed(0.75); setKeepPitch(false); }}>Slowed</button>
+                <button class="sr-btn" onClick={() => { setSpeed(1.25); setKeepPitch(true); }}>Nightcore</button>
+                <button class="sr-btn sr-btn--danger" onClick={() => { setSpeed(1.0); setKeepPitch(false); }}>Reset</button>
+              </div>
+              
+              <div class="sr-row">
```

ESLint

[error] Insert `⏎··`

(prettier/prettier)